<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<<<<<<< HEAD
<testsuite hostname="BLLTESTGOPAL" failures="0" tests="4" name="com.advaita.EndtoEnd.run.test.DataSetUpEndToEnd" time="133.916" errors="0" timestamp="2025-03-20T16:18:30 IST" skipped="0">
  <testcase classname="com.advaita.EndtoEnd.run.test.DataSetUpEndToEnd" name="verifyProcessCreate" time="13.724"/>
  <testcase classname="com.advaita.EndtoEnd.run.test.DataSetUpEndToEnd" name="verifyAutoGenerateQuestionCreateNewDatasetWithSpecifyingType" time="81.634"/>
  <testcase classname="com.advaita.EndtoEnd.run.test.DataSetUpEndToEnd" name="verifynewCreateMetaData" time="14.818"/>
  <testcase classname="com.advaita.EndtoEnd.run.test.DataSetUpEndToEnd" name="verifyCreateManualUpload" time="23.740"/>
=======
<<<<<<< HEAD
<<<<<<< HEAD
<testsuite hostname="BLLWYZABHIJITH" failures="0" tests="3" name="com.advaita.EndtoEnd.run.test.DataSetUpEndToEnd" time="68.804" errors="1" timestamp="2024-09-20T11:51:39 IST" skipped="0">
  <testcase classname="com.advaita.EndtoEnd.run.test.DataSetUpEndToEnd" name="verifyManualCreateNewDataset" time="33.352"/>
  <testcase classname="com.advaita.EndtoEnd.run.test.DataSetUpEndToEnd" name="verifynewCreateMetaData" time="17.825"/>
  <testcase classname="com.advaita.EndtoEnd.run.test.DataSetUpEndToEnd" name="verifyCreateManualUpload" time="17.627">
    <error message="Expected condition failed: waiting for visibility of Proxy element for: DefaultElementLocator &#039;By.xpath: //span[normalize-space()=&#039;Manual Data has been created successfully&#039;]&#039; (tried for 1 second(s) with 500 milliseconds interval)" type="org.openqa.selenium.TimeoutException">
      <![CDATA[org.openqa.selenium.TimeoutException: Expected condition failed: waiting for visibility of Proxy element for: DefaultElementLocator 'By.xpath: //span[normalize-space()='Manual Data has been created successfully']' (tried for 1 second(s) with 500 milliseconds interval)
at org.openqa.selenium.support.ui.WebDriverWait.timeoutException(WebDriverWait.java:84)
at org.openqa.selenium.support.ui.FluentWait.until(FluentWait.java:228)
at com.advaita.DataSetUp.PageObject.ManualUpload.createButtonAndConfirmation(ManualUpload.java:270)
at com.advaita.EndtoEnd.run.test.DataSetUpEndToEnd.verifyCreateManualUpload(DataSetUpEndToEnd.java:140)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:686)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:992)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:739)
at org.testng.TestRunner.run(TestRunner.java:614)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
Caused by: org.openqa.selenium.NoSuchElementException: no such element: Unable to locate element: {"method":"xpath","selector":"//span[normalize-space()='Manual Data has been created successfully']"}
  (Session info: chrome=129.0.6668.58)
For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
Build info: version: '4.23.0', revision: '4df0a231af'
System info: os.name: 'Windows 11', os.arch: 'amd64', os.version: '10.0', java.version: '17.0.11'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Command: [fdac38143cdf3f0df0413ab46781a853, findElement {value=//span[normalize-space()='Manual Data has been created successfully'], using=xpath}]
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 129.0.6668.58, chrome: {chromedriverVersion: 129.0.6668.58 (81a06fb873a9..., userDataDir: C:\Users\W2378\AppData\Loca...}, fedcm:accounts: true, goog:chromeOptions: {debuggerAddress: localhost:51801}, networkConnectionEnabled: false, pageLoadStrategy: normal, platformName: windows, proxy: Proxy(), se:cdp: ws://localhost:51801/devtoo..., se:cdpVersion: 129.0.6668.58, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:extension:minPinLength: true, webauthn:extension:prf: true, webauthn:virtualAuthenticators: true}
Session ID: fdac38143cdf3f0df0413ab46781a853
at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:499)
at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:480)
=======
<testsuite hostname="BLLTESTGOPAL" failures="0" tests="3" name="com.advaita.EndtoEnd.run.test.DataSetUpEndToEnd" time="123.196" errors="1" timestamp="2025-01-22T14:44:39 IST" skipped="0">
  <testcase classname="com.advaita.EndtoEnd.run.test.DataSetUpEndToEnd" name="verifyAutoGenerateQuestionCreateNewDatasetWithSpecifyingType" time="83.230"/>
  <testcase classname="com.advaita.EndtoEnd.run.test.DataSetUpEndToEnd" name="verifynewCreateMetaData" time="18.412"/>
  <testcase classname="com.advaita.EndtoEnd.run.test.DataSetUpEndToEnd" name="verifyCreateManualUpload" time="21.554">
    <error message="no such element: Unable to locate element: {&quot;method&quot;:&quot;xpath&quot;,&quot;selector&quot;:&quot;//tbody/tr[1]//td[normalize-space()=&#039;EmployeeC Details&#039;]&quot;}
  (Session info: chrome=131.0.6778.265)
For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
Build info: version: &#039;4.25.0&#039;, revision: &#039;8a8aea2337&#039;
System info: os.name: &#039;Windows 11&#039;, os.arch: &#039;amd64&#039;, os.version: &#039;10.0&#039;, java.version: &#039;23&#039;
Driver info: org.openqa.selenium.chrome.ChromeDriver
Command: [a3439f7c027567b34c3a7bc3e71903df, findElement {using=xpath, value=//tbody/tr[1]//td[normalize-space()=&#039;EmployeeC Details&#039;]}]
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 131.0.6778.265, chrome: {chromedriverVersion: 131.0.6778.264 (2d05e315153..., userDataDir: C:\Users\w2129\AppData\Loca...}, fedcm:accounts: true, goog:chromeOptions: {debuggerAddress: localhost:57377}, networkConnectionEnabled: false, pageLoadStrategy: normal, platformName: windows, proxy: Proxy(), se:cdp: ws://localhost:57377/devtoo..., se:cdpVersion: 131.0.6778.265, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:extension:minPinLength: true, webauthn:extension:prf: true, webauthn:virtualAuthenticators: true}
Session ID: a3439f7c027567b34c3a7bc3e71903df" type="org.openqa.selenium.NoSuchElementException">
      <![CDATA[org.openqa.selenium.NoSuchElementException: no such element: Unable to locate element: {"method":"xpath","selector":"//tbody/tr[1]//td[normalize-space()='EmployeeC Details']"}
  (Session info: chrome=131.0.6778.265)
For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
Build info: version: '4.25.0', revision: '8a8aea2337'
System info: os.name: 'Windows 11', os.arch: 'amd64', os.version: '10.0', java.version: '23'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Command: [a3439f7c027567b34c3a7bc3e71903df, findElement {using=xpath, value=//tbody/tr[1]//td[normalize-space()='EmployeeC Details']}]
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 131.0.6778.265, chrome: {chromedriverVersion: 131.0.6778.264 (2d05e315153..., userDataDir: C:\Users\w2129\AppData\Loca...}, fedcm:accounts: true, goog:chromeOptions: {debuggerAddress: localhost:57377}, networkConnectionEnabled: false, pageLoadStrategy: normal, platformName: windows, proxy: Proxy(), se:cdp: ws://localhost:57377/devtoo..., se:cdpVersion: 131.0.6778.265, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:extension:minPinLength: true, webauthn:extension:prf: true, webauthn:virtualAuthenticators: true}
Session ID: a3439f7c027567b34c3a7bc3e71903df
at java.base/jdk.internal.reflect.DirectConstructorHandleAccessor.newInstance(DirectConstructorHandleAccessor.java:62)
at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:501)
at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:485)
>>>>>>> ea653bb21122a5d6618e216dd0e951a8c4f0e483
at org.openqa.selenium.remote.ErrorCodec.decode(ErrorCodec.java:167)
at org.openqa.selenium.remote.codec.w3c.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:138)
at org.openqa.selenium.remote.codec.w3c.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:50)
at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:190)
at org.openqa.selenium.remote.service.DriverCommandExecutor.invokeExecute(DriverCommandExecutor.java:216)
at org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:174)
<<<<<<< HEAD
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:527)
at org.openqa.selenium.remote.ElementLocation$ElementFinder$2.findElement(ElementLocation.java:165)
at org.openqa.selenium.remote.ElementLocation.findElement(ElementLocation.java:59)
at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:357)
at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:351)
at org.openqa.selenium.support.pagefactory.DefaultElementLocator.findElement(DefaultElementLocator.java:68)
at org.openqa.selenium.support.pagefactory.internal.LocatingElementHandler.invoke(LocatingElementHandler.java:38)
at jdk.proxy2/jdk.proxy2.$Proxy17.isDisplayed(Unknown Source)
at org.openqa.selenium.support.ui.ExpectedConditions.elementIfVisible(ExpectedConditions.java:304)
at org.openqa.selenium.support.ui.ExpectedConditions$10.apply(ExpectedConditions.java:290)
at org.openqa.selenium.support.ui.ExpectedConditions$10.apply(ExpectedConditions.java:287)
at org.openqa.selenium.support.ui.FluentWait.until(FluentWait.java:203)
... 30 more
]]>
    </error>
  </testcase> <!-- verifyCreateManualUpload -->
=======
<testsuite hostname="BLLGOPALREDDY" failures="1" tests="3" name="com.advaita.EndtoEnd.run.test.DataSetUpEndToEnd" time="105.532" errors="0" timestamp="2024-09-09T13:28:01 IST" skipped="0">
  <testcase classname="com.advaita.EndtoEnd.run.test.DataSetUpEndToEnd" name="verifyAutoGenerateQuestionCreateNewDatasetWithSpecifyingType" time="64.677"/>
  <system-out/>
  <testcase classname="com.advaita.EndtoEnd.run.test.DataSetUpEndToEnd" name="verifynewCreateMetaData" time="20.194">
    <failure message="edit button is displayed. expected [false] but found [true]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: edit button is displayed. expected [false] but found [true]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertFalse(Assert.java:67)
at com.advaita.DataSetUp.PageObject.MetaData.verifyExecuteUpadtedMetaData(MetaData.java:899)
at com.advaita.EndtoEnd.run.test.DataSetUpEndToEnd.verifynewCreateMetaData(DataSetUpEndToEnd.java:110)
=======
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:545)
at org.openqa.selenium.remote.ElementLocation$ElementFinder$2.findElement(ElementLocation.java:165)
at org.openqa.selenium.remote.ElementLocation.findElement(ElementLocation.java:59)
at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:368)
at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:362)
at com.advaita.DataSetUp.PageObject.ManualUpload.valiadtionsAfterCreationOfManualUpload(ManualUpload.java:308)
at com.advaita.EndtoEnd.run.test.DataSetUpEndToEnd.verifyCreateManualUpload(DataSetUpEndToEnd.java:127)
>>>>>>> ea653bb21122a5d6618e216dd0e951a8c4f0e483
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:686)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:992)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
at org.testng.TestRunner.privateRun(TestRunner.java:739)
at org.testng.TestRunner.run(TestRunner.java:614)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
<<<<<<< HEAD
    </failure>
  </testcase> <!-- verifynewCreateMetaData -->
  <system-out/>
  <testcase classname="com.advaita.EndtoEnd.run.test.DataSetUpEndToEnd" name="verifyCreateManualUpload" time="20.661"/>
  <system-out/>
>>>>>>> 687a69d7ff80c98d2de02db845d9b8d8de698fd4
=======
    </error>
  </testcase> <!-- verifyCreateManualUpload -->
>>>>>>> ea653bb21122a5d6618e216dd0e951a8c4f0e483
>>>>>>> 02d29c0148f104b33822e240db974919af06f0b4
</testsuite> <!-- com.advaita.EndtoEnd.run.test.DataSetUpEndToEnd -->
